### PR TITLE
Add Openbank

### PR DESCRIPTION
**Pull request by Victor-Gomez**
Added Openbank.es

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -715,6 +715,13 @@ websites:
       facebook: nubankbrasil
       lang: pt
 
+    - name: Openbank
+      url: https://www.openbank.es
+      img: openbank.png
+      tfa: No
+      twitter: openbank_es
+      facebook: openbank.es
+
```
Comment (mxxcon) [inline _data/banking.yml:725]: Since this site is in Spanish, we should add this.
```suggestion
      lang: es

```

**Review by mxxcon — DISMISSED** [review]

**Comment by Victor-Gomez** [comment]
Updated as requested. Sorry for the delay, I was busy irl.

File: _data/banking.yml [hunk 2]
```diff
@@ -715,6 +715,15 @@ websites:
...
       lang: pt
 
+    - name: Openbank
+      url: https://www.openbank.es
+      img: openbank.png
+      tfa: No
+      twitter: openbank_es
+      facebook: openbank.es
+      lang: es
+      lang: es
```
Comment (jtagcat) [inline _data/banking.yml]:
```suggestion
```

**Review by jtagcat — CHANGES_REQUESTED** [review]
double lang:es

File: _data/banking.yml (hunk 2 above)
Comment (kmpoppe) [reply _data/banking.yml]: Resolved by reverting the addition of the double `lang`.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**